lib/broccoli/ember-app.js

```diff
@@ -64,13 +65,9 @@ module.exports = EmberApp;
   @param {Object} options Configuration options
 */
 function EmberApp() {
-  var args = [];
+  var args = map(arguments, function(item) { return item });
```

Comment:
this would deopt the function (not that it really matters, but it sorta hurts my soul...)